web-services/query/src/main/java/datawave/webservice/query/logic/composite/CompositeQueryLogicTransformer.java

```diff
@@ -75,33 +63,41 @@ public List<Object> readFromCache(List<CacheableQueryRow> row) {
     
     @Override
     public BaseQueryResponse createResponse(ResultsPage resultList) {
-        BaseQueryResponse result = null;
+        Exception lastFailure = null;
```

Comment:
It appears that we were calling all of the delegates which did not make sense.  Only the first one is required.  Same for the method below.